Libs/Reflex/src/main/java/reflex/AbstractReflexDataHandler.java

```diff
@@ -78,10 +79,10 @@ public ReflexValue pullData(RaptureURI uri) {
                 String contentType = blobContainer.getHeaders().get("Content-Type");
                 if (contentType != null) {
                     if (contentType.startsWith(CSV)) {
-                        List<ReflexValue> linesArray = new ArrayList<ReflexValue>();
+                        List<ReflexValue> linesArray = new ArrayList<>();
```

Comment:
It was done automatically. 
I turned on the 'format on save' option in Eclipse. It corrects the formatting to the house style.
As a side-effect it does other things like adding any missing annotations (@Override) and applying the Diamond operator. There's probably a way to turn it off.